featurebyte/query_graph/sql/interpreter/preview.py

```diff
@@ -174,16 +240,12 @@ def _get_query_graph_with_sample_on_primary_input_nodes(
         if not sample_on_primary_table:
             return query_graph, input_node_sampled
```

Comment:
When `input_node_sampled` flag is True, should we update the final sql to have a `LIMIT` clause to account for oversampling?

Reply:
Good catch! Will fix this accordingly.